<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>741</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLineEdit" name="URL_line">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>jenkins URL  :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_line">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passW_line">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>200</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>350</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="jobName_line">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>260</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="jobName_lbl">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>jobName</string>
     </property>
    </widget>
    <widget class="QLabel" name="status_lbl">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>480</y>
       <width>551</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>4</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="selectBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>select project</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
